tools/events-automation/state_territory_to_abbrev.py

```diff
@@ -0,0 +1,102 @@
+# Returns the abbreviated version of state/territory name for AU, CA, and US.
+# Information from Wikipedia.
+
+US_STATE_TO_ABBREV = {
```

Comment:
I would add a TODO into the doc (as a text or as a markdown comment) if this hardcoded list is going to be replaced by a function call a the future PR